### PR TITLE
Terrain Generation Base

### PR DESCRIPTION
**Pull request by trundler-dev**
We should do an analysis of these changes and figure out what we need to make this functional for a basic demo.

### CODE REVIEW

**Comment by ShadowBrian** [comment]
pog

File: code/Pawn/Worm.cs [hunk 1]
```diff
@@ -85,6 +85,23 @@ public override void Simulate( Client cl )
 				return;
 			}
 
+			if ( IsServer )
+			{
+				using ( Prediction.Off() )
+				{
+					int bonker = Input.Pressed( InputButton.Attack1 ) ? 1 : (Input.Pressed( InputButton.Attack2 ) ? 0 : -1);
```
Comment (xezno) [inline code/Pawn/Worm.cs:92]: What is bonker

File: code/Terrain/EdgeVertex.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿#region Assembly Sandbox.Game, Version=1.0.1.0, Culture=neutral, PublicKeyToken=null
```
Comment (xezno) [inline code/Terrain/EdgeVertex.cs:1]: You left in the decompile headers

**Review by xezno — CHANGES_REQUESTED** [review]

**Comment by trundler-dev** [comment]
Changes look good. Tomorrow I'll probably go through and start creating some issues for things we need to do (such as updating player controller, spawning, update test map).

**Review by xezno — APPROVED** [review]
LGTM

**Pull request merged**